apps/frontend/src/libs/components/menu/menu.tsx

```diff
@@ -7,13 +7,15 @@ type Properties = {
 	isOpened: boolean;
 	onClose: () => void;
 	onOpen: () => void;
+	usePositioning?: boolean;
```

Comment:
```suggestion
	usePositioning?: boolean;
```
`use` is a better prefix for functions. use `has` instead

Reply:
Maybe `enablePositioning` is better than `hasPositioning`? Has sounds like it's already active, while enable makes it clear we’re deciding to turn it on or not.

Reply:
prefer `has`/`is`